src/NUnitFramework/tests/Syntax/ThrowsTests.cs

```diff
@@ -153,5 +192,6 @@ public MyClass(string s)
                     throw new ArgumentNullException();
             }
         }
+
```

Comment:
ℹ It's too late to fix now unless you amend the same commit that introduced this. For the future, it's good practice to review the diff in Visual Studio or the git CLI before each commit to catch stuff like this. Makes for a weird git blame history.

Reply:
If we squash merge then it would be ok to revert this with a separate commit?

Reply:
Happy to do this if you want?

Reply:
True, yes!

Reply:
@ZenonBuratta Please do 👍. Not that this is a major problem, especially in the end of a test class, but still nice to have the history as clean as possible.